CONTRIBUTING.md

```diff
@@ -0,0 +1,172 @@
+# LaTeX Standards
+
+## Content
+
+### Sections on Algorithms and Data Structures
+
+The following subsections __must__ be present before a section will be considered for inclusion:
+
+- Description
+- Reference (only for data structures)
+- Applications
+- Sample Problem
+    - Input Format
+    - Sample Input
+    - Output Format
+    - Sample Output
+    - 'Lessons Learned'
+
+#### Description
+
+All data structures and algorithms should include a description section detailing:
+- what it does
+- the problem(s) that it solves
+- how it works
+- construction and run times in O-notation
+
+#### Reference
+
+If the section is for a data structure, it should include a subsection containing skeleton reference code.
+The reference code should include only what is necessary to use the data structure and perhaps usage examples in a main function if its use is not obvious enough.
+
+#### Applications
+
+A list of uses for the algorithm or data structure.
+For example, one application of a maximum flow algorithm is 'finding a minimum set of edges to disconnect a source and sink'.
+
+#### Sample Problem
+
+A problem whose solution utilizes the algorithm or data structure the section describes.
+Each problem should include a description that _clearly_ describes the problem. Following the problem description should be as follows:
+
+##### Input Format
+
+A precise description of the input that will be supplied to the code that will solve the problem described.
+
+##### Sample Input
+
+An example input that fully demonstrates and conforms to the format described in 'Input Format'.
+Use the `\acmlisting` command to insert your sample input.
+See the 'Code Listings' 
+
+##### Output Format
+
+A precise description of the output that should be produced from the code that will solve the problem described.
+
+##### Sample Output
+
+An example input that conforms to the format described in 'Output Format'.
+Use the `\acmlisting` command to insert your sample output.
+See the 'Code Listings' section under 'Format' for more information.
+
+##### Sample Solution
+
+A solution for the problem described.
+All solutions should be written in C++ __before__ providing solutions in other languages.
+
+##### Lessons Learned
+
+An unordered list of at least one non-trivial observation you had while writing the solution to your sample problem.
+
+***
+
+### Figures and Images
+
+If adding a visual to a section would greatly aid in the understanding of the topic, doing so is highly encouraged.
+Submissions should preferably be in the Encapsulated PostScript format.
+This provides very clear visuals, even when scaled up or down.
+Do _not_ simply pull images from the Internet.
+Doing so may violate the Hack Pack licensing terms.
+Any submission found to be doing so will be rejected until the appropriate changes are made.
+
+### Indexing
+
```

Comment:
We should also mention that if a term is already in the index because it is marked in another section, it should also be marked for indexing in yours.